package.json

```diff
@@ -48,5 +48,18 @@
     "node-fetch": "^1.7.2",
     "pouchdb": "^6.4.3",
     "ws": "^4.1.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PolymerLabs/arcs.git"
+  },
+  "author": "The Polymer Project Authors",
+  "license": "BSD-3-Clause",
+  "homepage": "https://github.com/PolymerLabs/arcs",
+  "bugs": {
+    "url": "https://github.com/PolymerLabs/arcs/issues"
+  },
+  "engines": {
+    "node": ">= 9.4.0"
```

Comment:
We had logic somewhere else checking this and alerting user if their version was insufficient, if this starts enforcing it maybe we can remove that logic? I think it's here: https://github.com/PolymerLabs/arcs/blob/master/tools/sigh.js#L99

Reply:
Digging into this a little deeper reveals that the engines property is used by the package consumer, it's informational only.  However it does make a good source of truth. So I adjusted the semvar checks to use the canonical versions there.